instance/api/openedx_appserver.py

```diff
@@ -100,7 +100,12 @@ def make_active(self, request, pk):
             return Response(
                 {"error": "Cannot make an unhealthy app server active."}, status=status.HTTP_400_BAD_REQUEST
             )
-        make_appserver_active(app_server.pk)
+        successfully_activated = make_appserver_active(app_server.pk)
```

Comment:
Similar to my comment earlier, we can move this into the previous `if` check to decrease repeat code, i.e.

```python
if not app_server.status.is_healthy_state or not make_appserver_active(app_server.pk):
         return Response(
             {"error": "Cannot make an unhealthy app server active."}, status=status.HTTP_400_BAD_REQUEST
         )
```

This'll make it active if the appserver is healthy and move on if successful, otherwise fail with the same error.